DscResourceCommonTests/Meta.Tests.ps1

```diff
@@ -26,7 +26,7 @@ Import-Module -Name (Join-Path -Path $TestHelper -ChildPath 'TestHelper.psm1') -
 $ErrorActionPreference = 'stop'
 Set-StrictMode -Version latest
 
-$RepoRoot = (Resolve-Path $PSScriptRoot\..).Path
+$RepoRoot = Split-Path -Parent (Split-Path -Parent $PSScriptRoot.Path)
```

Comment:
could you update me why we are making this change?
Also, I think we should use $Parent to get repoRoot